### PR TITLE
Include CHANGELOG in fix-whitespace and whitespace fixes

### PR DESCRIPTION
**Pull request by MatthewDaggitt**
As prompted by @andreasabel. There's no reason for the CHANGELOG not to be under whitespace fixes.

### CODE REVIEW

File: fix-whitespace.yaml [hunk 1]
```diff
@@ -7,5 +7,4 @@ included-files:
   - "*.md"
 
 excluded-files:
-  - "README/Text/Tabular.agda"
-  - CHANGELOG.md
```
Comment (jamesmckinna) [inline fix-whitespace.yaml:11]: AFAICT, this exclusion was added in commit by e0bd51a8c487075994a681309b62078328976adb
by @MatthewDaggitt but not sure why!?

File: fix-whitespace.yaml [hunk 2]
```diff
@@ -7,5 +7,4 @@ included-files:
...
+  - "README/Text/Tabular.agda"
```
Comment (jamesmckinna) [inline fix-whitespace.yaml:10]: Agre that `CHANGELOG` should fall under `fix-whitespace`

**Review by jamesmckinna — APPROVED** [review]
Happy to restore `CHANGELOG` under `fix-whitespace` ... not sure quite why it ever got removed.

What I don't understand is how all the other whitespace violations (including some of my own!) crept in in `*.agda` modules!?

Good clean-up work @MatthewDaggitt Thanks!

File: fix-whitespace.yaml (hunk 1 above)
Comment (JacquesCarette) [reply fix-whitespace.yaml:11]: Weirdly the PR that it is attached to points to commit [168f7e3](https://github.com/agda/agda-stdlib/pull/1379/commits/168f7e3d1cdbb0f7c97072f5bc58f87679dc2831) instead. It looks like this just 'snuck in' without discussion.

**Comment by andreasabel** [comment]
> What I don't understand is how all the other whitespace violations (including some of my own!) crept in in `*.agda` modules!?

Probably because the whitespace check isn't run by CI.  (Or not anymore, dunno.)

Here is the whitespace watchdog now:
- #2328

**Comment by andreasabel** [comment]
> There's no reason for the CHANGELOG not to be under whitespace fixes.

Technically, trailing whitespace in `.md` files stands for `<br/>`, but I think this is a misfeature that shouldn't be used.  (Use `<br/>` directly if you must.)
Another reason I can think of that folks want to use tabs, but again, one can instead use spaces.

**Review by andreasabel — APPROVED** [review]

**Pull request merged**